udf/allowed_histograms.sql

```diff
@@ -0,0 +1,259 @@
+      payload.processes.content.histograms.pwmgr_login_page_safety,
+      payload.processes.content.histograms.pwmgr_manage_copied_password,
+      payload.processes.content.histograms.pwmgr_manage_copied_username,
+      payload.processes.content.histograms.pwmgr_manage_deleted,
+      payload.processes.content.histograms.pwmgr_manage_opened,
+      payload.processes.content.keyed_histograms.sandbox_rejected_syscalls,
+      payload.processes.content.histograms.ssl_handshake_result,
+      payload.processes.content.histograms.ssl_handshake_version,
+      payload.processes.content.histograms.ssl_tls12_intolerance_reason_pre,
+      payload.processes.content.histograms.ssl_tls13_intolerance_reason_pre,
+      payload.processes.content.histograms.time_to_dom_complete_ms,
+      payload.processes.content.histograms.time_to_dom_content_loaded_end_ms,
+      payload.processes.content.histograms.time_to_dom_content_loaded_start_ms,
+      payload.processes.content.histograms.time_to_dom_interactive_ms,
+      payload.processes.content.histograms.time_to_dom_loading_ms,
+      payload.processes.content.histograms.time_to_first_click_ms,
+      payload.processes.content.histograms.time_to_first_interaction_ms,
+      payload.processes.content.histograms.time_to_first_key_input_ms,
+      payload.processes.content.histograms.time_to_first_mouse_move_ms,
+      payload.processes.content.histograms.time_to_first_scroll_ms,
+      payload.processes.content.histograms.time_to_load_event_end_ms,
+      payload.processes.content.histograms.time_to_load_event_start_ms,
+      payload.processes.content.histograms.time_to_non_blank_paint_ms,
+      payload.processes.content.histograms.time_to_response_start_ms,
+      payload.processes.content.histograms.webext_content_script_injection_ms,
+      payload.processes.content.histograms.webext_storage_local_get_ms,
+      payload.processes.content.histograms.webext_storage_local_set_ms,
+      payload.processes.content.histograms.webvr_time_spent_viewing_in_2d,
+      payload.processes.content.histograms.webvr_users_view_in
+    ) AS content,
+    STRUCT(
+--    payload.processes.gpu.histograms.checkerboard_severity,
+--    payload.processes.gpu.histograms.composite_time,
+--    payload.processes.gpu.histograms.content_frame_time,
+--    payload.processes.gpu.histograms.gpu_process_initialization_time_ms,
+--    payload.processes.gpu.histograms.webvr_time_spent_viewing_in_oculus,
+--    payload.processes.gpu.histograms.webvr_time_spent_viewing_in_openvr,
+--    payload.processes.gpu.histograms.webvr_users_view_in,
+      payload.processes.gpu.keyed_histograms.ipc_read_main_thread_latency_ms
+    ) AS gpu,
+    STRUCT(
+      payload.histograms.a11y_consumers,
+      payload.histograms.a11y_instantiated_flag,
+      payload.histograms.cert_validation_success_by_ca,
+      payload.histograms.checkerboard_severity,
+      payload.histograms.composite_time,
+--    payload.histograms.content_frame_time,
```

Comment:
@washort there are ten histograms here commented out that are not in the main ping schema. could determine whether these are in additional_properties, and if they are fill them in like so (and uncomment handling them downstream):
```suggestion
      JSON_EXTRACT(additional_attributes, '$.payload.histograms.CONTENT_FRAME_TIME') AS content_frame_time,
```